src/binance.mjs

```diff
@@ -41,6 +41,11 @@ export const BinanceCommands = {
         limitType:  'REQUESTS',
         reqAuth:    true,
         reqParams:  ['timestamp']
+    },
+    available: {
+        name: 'apiAvailable',
+        url: 'api/v1/ping',
+        weight: 1//Not sure what this is yet.
```

Comment:
Why are we tracking this? I am sure there is a reason but their api docs are so bad. You using https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md?

It looks like if we receive a 429 we should slow down instead? No point in the bot tracking this imo.

Reply:
Yeah using those docs. You get the 429 once you've exceeded and if you don't stop you get banned for progressively longer periods of time. The API has limits so it's more robust to take note of that ahead of time rather than reacting to the fact you've reached the limit - especially because the only way to react is to stop and that may cost you money.

Reply:
Even looking at their documentation it doesnt say how many times x can be done... it simply warns you when you hit their limits. I am not sure the purpose of recording weight at the moment unless you can find something in the documentation.

Reply:
Oh its in exchangeInfo. Nevermind.